src/Microsoft.AspNet.Razor.Runtime/TagHelpers/HtmlAttributeNameAttribute.cs

```diff
@@ -14,22 +14,39 @@ public sealed class HtmlAttributeNameAttribute : Attribute
         private string _dictionaryAttributePrefix;
 
         /// <summary>
+        /// Instantiates a new instance of the <see cref="HtmlAttributeNameAttribute"/> class with <c>Name==null</c>.
```

Comment:
`with <see cref="Name" /> set to <c>null</c>.` (so it can be refactored effectively).